pkg/addons/adminconsole/adminconsole.go

```diff
@@ -110,7 +110,7 @@ func (a *AdminConsole) Version() (map[string]string, error) {
 }
 
 func (a *AdminConsole) Name() string {
-	return "AdminConsole"
+	return releaseName
 }
```

Comment:
I have a feeling this is going to break something, let's see the version test

Reply:
surprisingly it didn't break unit tests!

Reply:
restored anyways.